### PR TITLE
Translate "Ruby 3.4.0 preview2 Released" (ko)

### PR DESCRIPTION
**Pull request by riseshia**
:link: https://github.com/ruby/www.ruby-lang.org/issues/2818

Translate https://github.com/ruby/www.ruby-lang.org/blob/9411a3d73d6e99d4a95973eb1f91365d9d5ad3f6/en/news/_posts/2024-10-07-ruby-3-4-0-preview2-released.md

Actual diff: a4b35f3

### CODE REVIEW

File: ko/news/_posts/2024-10-07-ruby-3-4-0-preview2-released.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+
+
+
+## 호환성 문제
+
+주의: 기능 버그 수정은 포함되어 있지 않습니다.
+
+* 에러 메시지와 백트레이스의 출력 결과가 변경됩니다.
+  * 인용 시작 부분을 나타내던 백틱 대신 작은 따옴표를 사용합니다. [[Feature #16495]]
+  * 메서드 이름 앞에 클래스 이름을 출력합니다(클래스가 불변하는 이름을 가지고 있다면). [[Feature #19117]]
```
Comment (yous) [inline ko/news/_posts/2024-10-07-ruby-3-4-0-preview2-released.md:56]:
```suggestion
  * 메서드 이름 앞에 클래스 이름을 출력합니다(클래스가 불변하는 이름을 가지고 있는 경우만). [[Feature #19117]]
```

File: ko/news/_posts/2024-10-07-ruby-3-4-0-preview2-released.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+  * `Range#size`는 이제 범위가 순회 가능하지 않다면 `TypeError`를 던집니다. [[Misc #18984]]
+
+
+
+## 호환성 문제
+
+주의: 기능 버그 수정은 포함되어 있지 않습니다.
+
+* 에러 메시지와 백트레이스의 출력 결과가 변경됩니다.
+  * 인용 시작 부분을 나타내던 백틱 대신 작은 따옴표를 사용합니다. [[Feature #16495]]
```
Comment (yous) [inline ko/news/_posts/2024-10-07-ruby-3-4-0-preview2-released.md:55]: I think it's mentioning the difference between `` `foo' ``, `` `<main>' `` and `'foo'`, `'<main>'`. So how about "여는 따옴표" for "opening quote?"

```suggestion
  * 여는 따옴표로 백틱 대신 작은 따옴표를 사용합니다. [[Feature #16495]]
```

File: ko/news/_posts/2024-10-07-ruby-3-4-0-preview2-released.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+  * 다른 키는 `=>` 주변에 공백이 추가되었습니다. 예시: `'{"user" => 1}'`. 이전에는 그렇지 않았습니다. 예시: `'{"user"=>1}'`.
+
+## C API 변경
+
+* `rb_newobj`와 `rb_newobj_of`(그리고 대응하는 매크로인 `RB_NEWOBJ`, `RB_NEWOBJ_OF`, `NEWOBJ`, `NEWOBJ_OF`)가 삭제됩니다. [[Feature #20265]]
+* 폐기 예정인 `rb_gc_force_recycle` 함수가 삭제됩니다. [[Feature #18290]]
+
+## 구현 개선
+
+* `Array#each`를 더 나은 성능을 위해 Ruby로 재작성했습니다. [[Feature #20182]]
```
Comment (yous) [inline ko/news/_posts/2024-10-07-ruby-3-4-0-preview2-released.md:80]:
```suggestion
* 더 나은 성능을 위해 `Array#each`를 Ruby로 재작성했습니다. [[Feature #20182]]
```

File: ko/news/_posts/2024-10-07-ruby-3-4-0-preview2-released.md (hunk 1 above)
Comment (riseshia) [reply ko/news/_posts/2024-10-07-ruby-3-4-0-preview2-released.md]: fixed 9088ed4

File: ko/news/_posts/2024-10-07-ruby-3-4-0-preview2-released.md (hunk 2 above)
Comment (riseshia) [reply ko/news/_posts/2024-10-07-ruby-3-4-0-preview2-released.md]: fixed 9088ed4

File: ko/news/_posts/2024-10-07-ruby-3-4-0-preview2-released.md (hunk 3 above)
Comment (riseshia) [reply ko/news/_posts/2024-10-07-ruby-3-4-0-preview2-released.md]: fixed 9088ed4

**Review by yous — APPROVED** [review]
Thanks!

**Pull request merged**